dwave/cloud/client.py

```diff
@@ -696,36 +750,87 @@ def solvers(self, name=None, qpu=None, software=None, vfyc=None,
 
                 solver = client.solvers(online=True, name='solver[12]')[0]
                 solver = client.solvers(online=True, name='solver1|solver2')[0]
+                solver = client.solvers(online=True, chip_id__regex='solver[12]')[0]
+
+            Get all solvers that contain a coupler ``(0, 128)``::
+
+                solver = client.solvers(couplers__contains=[0, 128])
+
+            Get all solvers that support 1000 for ``num_reads``:
+
+                solver = client.solvers(num_reads_range__covers=1000)
```

Comment:
"covers" is not in the above list: "available, lt, lte, gt, gte, eq, within, contains, regex "

Reply:
The list is already fixed (in latest commit).